get_github_coderabbit_review/coderabbit_fetcher/async_processing/async_task_manager.py

```
@@ -0,0 +1,594 @@
+                if blocked_by_failed:
+                    # Tasks blocked by failed dependencies
+                    error_msg = "Tasks blocked by failed dependencies:"
+                    for task_id, dep_id, status in blocked_by_failed:
+                        error_msg += f"\n- {task_id} blocked by {dep_id} ({status.value})"
+                    logger.error(error_msg)
+                    raise ValueError(error_msg)
+                else:
+                    # True circular dependency
+                    unresolved = list(remaining_tasks.keys())
+                    logger.error(f"Circular dependency detected in tasks: {unresolved}")
+                    raise ValueError(f"Circular dependency detected in tasks: {unresolved}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)